frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -27,7 +27,6 @@ local MODE = {
     book_time_to_read = 6,
     chapter_time_to_read = 7,
     frontlight = 8,
-    mem_usage = 9,
```

Comment:
I am OK to keep it. But I would argue we can do this more smartly. We can add a restart KOReader function in UIManager, and use a plugin to track the memory usage. Once the memory usage reaches the limitation, an InfoMessage will be shown to ask whether the KOReader should be restarted.